messaging-activemq/src/main/java/org/wildfly/extension/messaging/activemq/BridgeDefinition.java

```diff
@@ -181,7 +181,7 @@ private BridgeDefinition() {
     public void registerAttributes(ManagementResourceRegistration registry) {
         for (AttributeDefinition attr : ATTRIBUTES) {
             if (!attr.getFlags().contains(AttributeAccess.Flag.STORAGE_RUNTIME)) {
-                registry.registerReadWriteAttribute(attr, null, BridgeWriteAttributeHandler.INSTANCE);
+                registry.registerReadWriteAttribute(attr, null, new ReloadRequiredWriteAttributeHandler(ATTRIBUTES));
```

Comment:
create new ReloadRequiredWriteAttributeHandler(ATTRIBUTES) outside the loop, to lower memory footprint and most of all GC pressure during boot.